fastparquet/util.py

```diff
@@ -26,7 +26,7 @@ class ParquetException(Exception):
 
 def sep_from_open(opener):
     if opener is default_open:
-        return os.sep
+        return '/'
```

Comment:
well, maybe this function can be collapsed.  But first let's see if this PR is good enough to be accepted.